core/src/main/java/org/mule/runtime/core/api/el/dataweave/DataWeaveArtifactContext.java

```diff
@@ -25,14 +24,33 @@
  *
  * @since 4.0
  */
-public class DataWeaveArtifactContext extends AbstractArtifactContext {
+public class DataWeaveArtifactContext {
```

Comment:
can't we expose an interface instead? or at most, make it priviledged

Reply:
DW doesn't handle interfaces, only concrete classes